tests/benchdnn/conv/conv.cpp

```diff
@@ -667,6 +668,47 @@ void check_known_skipped_case(const prb_t *prb, res_t *res) {
     if (prb->alg == WINO) {
         if (is_cpu()) {
 #if DNNL_CPU_RUNTIME != DNNL_RUNTIME_NONE
+#if DNNL_AARCH64
+            bool pad_ok_f32 = prb->pw <= 1 && prb->ph <= 1 && prb->pw_r <= 1
+                    && prb->ph_r <= 1;
+            bool shape_ok = prb->ndims == 4 && prb->g == 1 && prb->kh == 3
+                    && prb->kw == 3 && prb->sh == 1 && prb->sw == 1
+                    && prb->dh == 0 && prb->dw == 0 && !prb->has_groups
+                    && pad_ok_f32;
+
+            const bool dir_ok = (prb->dir & FLAG_FWD);
+
+            const auto &po = prb->attr.post_ops;
+
+            // "true" here stands for eltwise.scale == 1.f check
```

Comment:
Minor: AFAIR, this is general ACL (only?) limitation w.r.t. eltwise post-op. I would consider putting it under `check_eltwise_post_ops_aarch64_acl(res_t *res)` call and use it whenever needed. Putting it next to `check_known_skipped_case_common` sounds good to me.

If this limitation for ACL only, I'd wrap it under `DNNL_AARCH64_USE_ACL` to be safe.